go/src/koding/socialapi/presence/presence.go

```diff
@@ -0,0 +1,38 @@
+package presence
+
+import (
+	"errors"
+	"net/http"
+	"net/url"
+
+	"koding/socialapi"
+)
+
+type Client struct {
```

Comment:
To add api.SessionContextKey to http.Request.Context here:

https://github.com/koding/koding/pull/9851/files#diff-64803beca13535937999dd1daec4743cR22

This way single transport knows for which user it should get the ClientID.